airflow-core/docs/administration-and-deployment/plugins.rst

```diff
@@ -192,13 +194,28 @@ definitions in Airflow.
         "name": "Name of the Middleware",
     }
 
+    # Creating a iframe view that will be rendered in the Airflow UI.
+    iframe_view_with_metadata = {
+        "name": "Name of the Iframe View",
+        # Source URL of the iframe. This URL can be templated using context variables.
+        "src": "https://example.com/{DAG_ID}/{RUN_ID}/{TASK_ID}",
+        # Destination of the iframe view. This is used to determine where the iframe will be loaded in the UI.
+        # Supported locations are Literal["nav", "dag", "dag_run", "task", "task_instance"]
+        "destination": "dag_run",
+        # Optional icon, anything from Font Awesome can be used there.
+        "icon": "fa-globe",
```

Comment:
Maybe we don't want that but a plain "url", the constraint is it has to be hosted somewhere right ? (But we don't care about that,  there are plenty of ways to host static content)

Reply:
Modified to plain url to svg